src/components/responsive-video/responsive-video.vue

```diff
@@ -20,16 +21,36 @@
       <app-icon name="play" />
     </template>
   </vue-dato-video>
+  <video
+    v-else-if="props.gif"
+    class="responsive-video"
+    autoplay
+    controls
+    :style="{
+      aspectRatio: `${props.gif.width} / ${props.gif.height}`,
+    }"
+  >
+    <source
+      :src="`${props.gif.url}?fm=mp4`"
+      type="video/mp4"
+    >
+  </video>
```

Comment:
Hmm I don't know if this is the way to go, can't we use vue-dato-video for this?
Would like some more context, it's not really a gif right but a mp4 video that autoplays?

Reply:
I agree that, if possible, this should be formatted so you can use the vue-dato-video for both